### PR TITLE
docs: add link to interactive demo

### LINKED ISSUE #346: [DOC] Killercoda Demo
### What kind of documentation improvement is needed?

Other

### What is the change that is needed?

I'm creating this issue to document something I'd like to do. I think it would be helpful to have an interactive tutorial environment for Copa through [Killercoda](https://killercoda.com/). Notary does this, and I think it's a cool feature to have

### PR DESCRIPTION
**Pull request by salaxander**
Add a link to the newly created Killercoda interactive demo!

Closes #346

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@
 
 ![intro](demo/copa-demo.gif)
 
+### Interactive Demo
+You can also try out a fully interactive `copa` demo on [Killercoda](https://killercoda.com/salaxander/scenario/try-copa)
```
Comment (sozercan) [inline README.md:15]: do we want to move this to `project-copacetic` org?

**Comment by salaxander** [comment]
@sozercan - killercoda account name has been updated, this should be good to go now

**Comment by sozercan** [comment]
@salaxander can we add a CI for this to make sure it's working, and if not, we'll get some sort of signal in the future?

**Comment by sozercan** [comment]
@salaxander i meant this part of it: https://github.com/project-copacetic/copa-action/blob/main/action.yaml#L36-L41
This will automatically pull in the latest release instead of being hardcoded

**Comment by salaxander** [comment]
@sozercan I recently updated the script to pull in the latest release of Copa

**Comment by sozercan** [comment]
@salaxander can we add a simple nightly or weekly github action to execute the scripts so we'll know if it's broken

**Comment by sozercan** [comment]
Closing due to inactivity. Please feel free to re-open if there's ongoing work. Thanks!

**Pull request closed without merging**